app/src/main/res/values/strings.xml

```diff
@@ -32,7 +32,7 @@
     <string name="plant_date_header" translation_description="Text that is shown on each plant on the 'My Garden' screen which says the word Planted to show date of planting below">Planted</string>
     <string name="watered_date_header" translation_description="Text that is shown on each plant on the 'My Garden' screen which says the word Last Watered to show date of last watering below">Last Watered</string>
     <string name="menu_item_share_plant" translation_description="The label for the Share menu item">Share</string>
-    <string name="share_text_plant" translation_description="The default messaged used when sharing a plant on SNS, etc.">Check out the %s plant in the Android Sunflower app</string>
+    <string name="share_text_plant" translation_description="The default messaged used when sharing a plant on SMS, etc.">Check out the %1$s plant in the Android Sunflower app \n%2$s</string>
```

Comment:
I think it should be in another Pull Request as it is not related to topic :-)

Reply:
SNS is short for social networking service